### PR TITLE
Allow accessing the transformer

### PR DESCRIPTION
**Pull request by Mohammad-Alavi**
# Summary
Addded a public method to be able to access the transformer.

# Reasoning
## TL;DR
Using my own extended `Fractal` facade, this allows me to call the `ok` method both **with** and **without** setting a transformer.
* `Fractal::create($this->user, UserTransformer::class)->ok();`
* `Fractal::ok();`

Otherwise, I get a `NoTransformerSpecified` exception.

> Yes, I can just create a `200` response using a thousand other ways but I woud like to be consistence and use the `Fractal` facade to generate all of my responses.
 
### Background
I am using your awesome [larave-fractal](https://github.com/spatie/laravel-fractal/tree/main) package which has the [`Spatie\Fractal\Fractal`](https://github.com/spatie/laravel-fractal/blob/main/src/Fractal.php) that extends the [`Spatie\Fractalistic\Fractal`](https://github.com/spatie/fractalistic/blob/main/src/Fractal.php).

#### Glossary
| When I say | I mean |
|------------|--------|
| `LaravelFractal` | [`Spatie\Fractal\Fractal`](https://github.com/spatie/laravel-fractal/blob/main/src/Fractal.php) |
| `FractalisticFractal` | [`Spatie\Fractalistic\Fractal`](https://github.com/spatie/fractalistic/blob/main/src/Fractal.php) |
| `MyFractal` | My class that extends [`Spatie\Fractal\Fractal`](https://github.com/spatie/laravel-fractal/blob/main/src/Fractal.php) |

### Why
`LaravelFractal` class has a convenient `respond` method that helps with genarating responses.  
In my project I have added some other convenient methods to `MyFractal`. For example,
```php
class MyFractal extends LaravelFractal
{
    public function ok(): JsonResponse
    {
        // notice here, I am using the proposed method
        if (is_null($this->getTransformer())) {
            // `empty` is just another convenient method of mine.
            // It returns a transformer that returns an empty array, [], never mind it!
            $this->transformWith(Transformer::empty());
        }
    
        return $this->respond(200); // I am using the `LaravelFractal::respond`
    }
}
```
Using my own facade, this allows me to call the `ok` method both **with** and **without** setting a transformer.
* `MyFractal::create($this->user, UserTransformer::class)->ok();`
* `MyFractal::ok();`

Otherwise, I get a `NoTransformerSpecified` exception.
This happens because `MyFractal::ok` method uses the `LaravelFractal::respond` method underneath. And in turn,
[`LaravelFractal::respond`](https://github.com/spatie/laravel-fractal/blob/e9fee265f8e9988bc32b0a7641770a0b2e64590f/src/Fractal.php#L89) method calls the [`FractalisticFractal::createData`](https://github.com/spatie/fractalistic/blob/fdb6016dc566c96248f4157e48e95bb08ab06437/src/Fractal.php#L382) method and this method throws an exception if it is called without a transfromer. (which is the case with `MyFractal::ok` mthod)
```php
    public function createData()
    {
        if (is_null($this->transformer)) {
            throw new NoTransformerSpecified();
        }
        // ...
    }
```

### CODE REVIEW

File: src/Fractal.php [hunk 1]
```diff
@@ -21,7 +22,7 @@ class Fractal implements JsonSerializable
     /** @var string|\League\Fractal\Serializer\SerializerAbstract */
     protected $serializer;
 
-    /** @var string|callable|\League\Fractal\TransformerAbstract */
+    /** @var string|callable|\League\Fractal\TransformerAbstract|null */
```
Comment (Mohammad-Alavi) [inline src/Fractal.php:25]: Just update the outdated docblock, because the transformer can be `null` already.

**Comment by Nielsvanpach** [comment]
Thanks!
